server.js

```diff
@@ -144,6 +144,10 @@ app.get("/check", (req, res) => {
 
 });
 
+app.get('*', function(res){
+  return res.status(404).render('404');
+});
```

Comment:
utilisation des guillemets
```suggestion
app.get("*", function(res){
  return res.status(404).render("404");
});
```